### ISSUE TITLE
fetch in getInitialProps broken?

### ISSUE THREAD

**Issue by lipp** [zeit/next.js#891]
I used to do an isomorphic fetch in [`getInitialProps`](https://github.com/lipp/login-with/blob/master/example/nextjs/pages/token.js) from another sub-domain.

This worked some days ago.

However, instead of fetching the resource specified, the browser as of today tries to load some <pagename>.js which returns some "component" json. I guess this is a "prefetched" version of what the browser used to `fetch` "manually".

Bottom line: I don't get my props.

**Comment by lipp** [comment]
... verifying with previous build....

**Comment by lipp** [comment]
...sorry my mistake!
